microsoft/botframework-solutions
# [Virtual Assistant][TypeScript] Rename Botbuilder libraries

## Description

- Rename `bot-solution` to `botbuilder-solutions`
- Rename `bot-skill` to `botbuilder-skills`
- Update references of those dependencies 

Close #1079 

## Testing Steps
1. Go to `AI\solutions\Virtual-Assistant\src\typescript`
2. Check the name of the botbuilder libraries
3. Run `rush update`
4. Run `rush rebuild`
5. Check that all the projects are taking the references to the dependencies correctly

## Checklist
\-
